packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/views/explorer/services/api/IExplorerModificationService.java

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.application.views.explorer.services.api;
+
+import java.util.List;
+
+import org.eclipse.sirius.components.core.api.IEditingContext;
+import org.eclipse.sirius.components.representations.IStatus;
+import org.eclipse.sirius.components.trees.Tree;
+
+/**
+ * Service related to modification from Explorer.
+ *
+ * @author frouene
+ */
+public interface IExplorerModificationService {
```

Comment:
I think "modification" meaning is too wide.
It makes me think other kind of modification could be added to this interface but, IMO the implementation of this interface is big enough to add more behavior.
I would have changed this interface name to something more specific like `IExplorerObjectDropService`

Reply:
Maybe this interface is not needed, maybe put all the implementation in the `ExplorerDropTreeItemProvider` can be considered because there will only be one implementation of this service in the whole application and, it is provided in the same package as the `ExplorerDropTreeItemProvider`

Reply:
Yeah, I think you're right, otherwise the provider is a very basic service that does nothing but call the `IExplorerModificationService `